### DISCUSSION TITLE
AI とか文章の類似度みたいな最近のウケが良いっぽくて響きがいい何かを使って、今見ている記事と関連性が高い記事をレコメンドして欲しい

### DISCUSSION THREAD

**Discussion by blono** [increments/qiita-discussions#31] [要望・アイデア・不具合]
例えば、css で flex がどうとかいう記事を見ている時に、それと類似度が高い記事も目立つ形でレコメンドしてくれると嬉しいかもと思いました。
技術力アピールにもなりそうな予感がします。

**Comment by kskwtnk** [comment]
@blono 
ご意見ありがとうございます！

関連記事についてはあまり手を入れられておらず、そこまで類似性が高くない状態での提供になっています……🥲
手法は全然定まっていませんが、記事を読むときの体験の一環として関連記事の精度を上げていこうというIssueは立っているため、取り組んでいきます！
